README.md

```diff
@@ -1,7 +1,18 @@
 # PocketEstate
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
-
+[![Java CI](https://github.com/AY2021S2-CS2103T-T13-4/tp/actions/workflows/gradle.yml/badge.svg)](https://github.com/AY2021S2-CS2103T-T13-4/tp/actions/workflows/gradle.yml)
 ![Ui](docs/images/Ui.png)
 
 PocketEstate enables easy organisation of mass clientele property information through sorting of information by price, location and housing type, that may otherwise be difficult to manage. 
+
+## User guide
+For more information on how to use PocketEstate, please refer to our [User Guide](./docs/UserGuide.md). 
+
+## Developer Guide
+For more information on the development of PocketEstate, please refer to our [Developer Guide](./docs/DeveloperGuide.md). 
+
+## About Us
+We are a team of 5 from the School of Computing, National University of Singapore. For more information, please refer to [About Us](./docs/AboutUs.md) . 
```

Comment:
Should we use the actual website links instead of the link to the markdown file?